Tests/eppo/ClientTests.swift

```diff
@@ -132,13 +158,13 @@ final class eppoClientTests: XCTestCase {
    override func setUpWithError() throws {
        try super.setUpWithError()
        loggerSpy = AssignmentLoggerSpy()
-       eppoClient = EppoClient("mock-api-key",
+       eppoClient = EppoClient(apiKey: "mock-api-key",
                                host: "http://localhost:4001",
                                assignmentLogger: loggerSpy.logger)
    }
    
    func testUnloadedClient() async throws {
-       XCTAssertThrowsError(try eppoClient.getStringAssignment("badFlagRising", "allocation-experiment-1"))
```

Comment:
Whoops - the flag and subject arguments are backwards here. labeled parameters and type-enforcement FTW.